Formula/bracken.rb

```diff
@@ -0,0 +1,28 @@
+class Bracken < Formula
+  # cite Lu_2017: "https://doi.org/10.7717/peerj-cs.104"
+  desc "Bayesian estimation species abundances from Kraken output"
+  homepage "https://ccb.jhu.edu/software/bracken/index.shtml"
+  url "https://github.com/jenniferlu717/Bracken/archive/v2.2.tar.gz"
+  sha256 "307495060d3575c3a90c87cf82074d4a63d20cf4062658f161951c60a66fd319"
+
+  depends_on "gcc" if OS.mac?
+
+  fails_with :clang # uses OpenMP
+
+  def install
+    cd "src" do
+      inreplace "Makefile", "g++", "$(CXX)"
+      system "make"
+    end
+
+    inreplace "bracken", "$DIR/src", libexec
+    inreplace "bracken-build", "$DIR/src", libexec
+
+    bin.install "bracken", "bracken-build"
+    libexec.install "src/est_abundance.py", "src/generate_kmer_distribution.py", "src/kmer2read_distr", "src/kreport2mpa.py"
+  end
+
+  test do
+    assert_match "Usage", shell_output("#{bin}/bracken -h 2> /dev/null")
```

Comment:
```suggestion
    assert_match "Usage", shell_output("#{bin}/bracken 2>&1")
```
Does removing the `-h` option work?

Reply:
No, unfortunately that's "worse":

```
 ❯ bracken
 >> Checking for Valid Options...
 ERROR: /database100mers.kmer_distrib does not exist
        Run bracken-build to generate the kmer distribution file.
```

versus

```
 ❯ bracken -h
/usr/local/bin/bracken: illegal option -- h
Usage: bracken -d MY_DB -i INPUT -o OUTPUT -r READ_LEN -l LEVEL -t THRESHOLD
  MY_DB          location of Kraken database
  INPUT          Kraken REPORT file to use for abundance estimation
  OUTPUT         file name for Bracken default output
  READ_LEN       read length to get all classifications for (default: 100)
  LEVEL          level to estimate abundance at [options: D,P,C,O,F,G,S] (default: S)
  THRESHOLD      number of reads required PRIOR to abundance estimation to perform reestimation (default: 0)
```

Altogether this doesn't make that much of a difference, but just looks like somewhat of a "hack".